server/models/task.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Task extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+      Task.belongsTo(models.User)
+    }
+  };
+  Task.init({
+    title: {
+      type: DataTypes.STRING,
+      validate: {
+        notEmpty: {
+          msg: "Please fill in the column"
+        }
+      }
+    },
+    description: {
+      type: DataTypes.TEXT,
+      validate: {
+        notEmpty: {
+          msg: "Please fill in the column"
+        }
+      }
+    },
+    category: {
+      type: DataTypes.STRING,
+      validate: {
+        notEmpty: {
+          msg: "Please fill in the column"
+        }
+      }
+    },
+    assigned_to: {
```

Comment:
assign to itu untuk user Id juga ya?

Reply:
Bukan kak, itu bisa diisi langsung sama user nanti. Jadi user kan bakal assign suatu task bisa ke diri sendiri atau temannya. Jadi cukup ketik nama aja.